docs/source/common_issues.rst

```diff
@@ -238,17 +246,18 @@ of a name (assume here that ``Shape`` is the base class of both
 
    shape = Circle()    # Infer shape to be Circle
    ...
-   shape = Triangle()  # Type error: Triangle is not a Circle
+   shape = Triangle()  # error: Incompatible types in assignment
+                       # (expression has type "Triangle", variable has
+                       # type "Circle")
 
-You can just give an explicit type for the variable in cases such the
-above example:
+You can give an explicit type for the variable in cases such the above example:
 
 .. code-block:: python
 
-   shape = Circle() # type: Shape   # The variable s can be any Shape,
-                                    # not just Circle
+   shape: Shape = Circle()  # The variable s can be any Shape,
```

Comment:
I agree we don't need to change existing docs with respect to PEP 526 syntax.
(Although, I think it is OK to use it from time to time in _new_ docs.)